webapi/src/main/twirl/org/knora/webapi/messages/twirl/queries/sparql/admin/isEntityUsed.scala.txt

```diff
@@ -22,7 +22,8 @@ PREFIX salsah-gui: <http://www.knora.org/ontology/salsah-gui#>
 SELECT DISTINCT ?isUsed
 
 WHERE {
-  	BIND(IRI("@entityIri") AS ?entity)
+    BIND(IRI("@entityIri") AS ?entity)
+    ?s ?p ?entity .
     BIND(true AS ?isUsed)
-	?s ?p ?entity
 }
+LIMIT 1
```

Comment:
this is a strange way of writing this query - I didn't even realise this works. Is this the thing where you wanted to employ ASK?

Reply:
Yes, closing this PR in favor of https://github.com/dasch-swiss/dsp-api/pull/2739